packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -48,13 +49,15 @@ class AppSizeController {
   /// The node set as the analysis tab root.
   ///
   /// Used to build the treemap and the tree table for the analysis tab.
-  ValueListenable<TreemapNode?> get analysisRoot => _analysisRoot;
-  final _analysisRoot = ValueNotifier<TreemapNode?>(null);
+  ValueListenable<Selection<TreemapNode>> get analysisRoot => _analysisRoot;
+  final _analysisRoot =
+      ValueNotifier<Selection<TreemapNode>>(Selection<TreemapNode>());
```

Comment:
if we need to make analysisRoot accessible from the tree table, we can just get rid of the public listenable and make the notifier public:
```
final analysisRoot =
      ValueNotifier<Selection<TreemapNode>>(Selection<TreemapNode>());
```

then you won't have to cast later when you use `analysisRoot` as the `selection` notifier